### PR TITLE
adding community pages

### LINKED ISSUE #738: Provide community page as part of website
Currently, the Keptn community page is a README on Github. Would be great to have it as part of the Keptn website.

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Closes #738 

Signed-off-by: jetzlstorfer <juergen.etzlstorfer@dynatrace.com>

### CODE REVIEW

**Comment by jetzlstorfer** [comment]
Preview: https://deploy-preview-742--keptn.netlify.app/community/

File: content/community/_index.md [hunk 1]
```diff
@@ -1,64 +1,55 @@
...
+## Slack 
 
-## Communication 
-
-### Slack 📟
-
-We have a dedicated [Keptn slack workspace](https://slack.keptn.sh) (for in-depth discussions and getting in touch with the team) as well as a #keptn channel in the CNCF Slack workspace. We are happy to talk to you!
+We have a dedicated [Keptn slack workspace](https://slack.keptn.sh) (for in-depth discussions and getting in touch with the team) as well as a #keptn channel in the [CNCF Slack workspace](http://slack.cncf.io/). We are happy to talk to you!
 
 ✅ Chat with other people about using Keptn
```
Comment (johannes-b) [inline content/community/_index.md:15]: Can you please change the icons to :heavy_check_mark: ` : heavy_check_mark : `

File: content/community/contributing/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_multicard: false
+icon: callout-idea
+---
+ 
+
+We are welcoming new contributors to the Keptn project!
+
+## Getting started
+
+1️⃣ If you are new to Keptn and want to get started contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```
Comment (johannes-b) [inline content/community/contributing/index.md:14]: Please don't use icons for numbering here. Go with: `1.` `2.`

File: content/community/_index.md (hunk 1 above)
Comment (jetzlstorfer) [reply content/community/_index.md:15]: we agreed to keep it as the heavy check mark is black on MacOS and not green

File: content/community/contributing/index.md (hunk 2 above)
Comment (jetzlstorfer) [reply content/community/contributing/index.md]: changed it to numbers

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**